gramps/gen/lib/placetype.py

```diff
@@ -19,22 +21,58 @@
 #
 
 """
-Provide the different place types.
+Place Type class for Gramps
 """
 
 #-------------------------------------------------------------------------
 #
 # Gramps modules
 #
 #-------------------------------------------------------------------------
-from .grampstype import GrampsType
+from .secondaryobj import SecondaryObject
+from .datebase import DateBase
+from .citationbase import CitationBase
+from .const import IDENTICAL, EQUAL, DIFFERENT
 from ..const import GRAMPS_LOCALE as glocale
 _ = glocale.translation.gettext
 
-class PlaceType(GrampsType):
 
-    UNKNOWN = -1
-    CUSTOM = 0
+#-------------------------------------------------------------------------
+#
+# Place Type
+#
+#-------------------------------------------------------------------------
+class PlaceType(SecondaryObject, CitationBase, DateBase):
+    """
+    Place Type class.
+
+    This class is for keeping information about place typess.
+    This class is similar to GrampsType based classes, but has other features.
+    It supports 3 ranges of types;
```

Comment:
After reading this, it seems like we are giving a lot of responsibility to a single value. Would it improve the clarity of the code if we added another member to keep track of what "kind" of type it represents?